src/ClusterSequence.jl

```diff
@@ -321,8 +321,10 @@ function exclusive_jets(clusterseq::ClusterSequence{U}; dcut = nothing, njets =
     end
 
     # Check that an algorithm was used that makes sense for exclusive jets
-    if !(clusterseq.algorithm ∈
-         (JetAlgorithm.CA, JetAlgorithm.Kt, JetAlgorithm.EEKt, JetAlgorithm.Durham))
+    if clusterseq.algorithm == JetAlgorithm.GenKt && clusterseq.power < 0
+        throw(ArgumentError("Algorithm GenKt requires power >= 0 (power=$(clusterseq.power))"))
+    elseif clusterseq.algorithm ∉
+           (JetAlgorithm.CA, JetAlgorithm.Kt, JetAlgorithm.EEKt, JetAlgorithm.Durham)
```

Comment:
Same logic flaw as below - GenKt with p >= 0 will fail this test and throw an error